### ISSUE TITLE
Alarm screen doesn't expand properly

### ISSUE THREAD

**Issue by esmoak** [fennifith/Alarmio#40]
## Device Info

- Brand: Huawei
- Model name: Mate SE, BND-L34
- Android version: stock, 8.0.0

## Steps to Reproduce

- create multiple alarms
- lowest alarm gets cut off on bottom
- plus sign usually covers second alarm delete button or alarm on/off toggle
- unable to expand alarm screen with an upwards slide, a tap, expanding first alarm, or any method. 

The alarm screen works fine initially but quits working after some time. 

Force stop and clear app data doesn't fix, but uninstall and reinstall does. 

Happened multiple times after reinstalling. 

Version 1.0.0 from Izzy's repo.

**Comment by fennifith** [comment]
I can't reproduce this on a stock emulator or my physical device. Could you possibly take a screen recording of this occurring? FYI, here is a recording of how the interaction is _supposed_ to work: [https://imgur.com/a/3imytkd](https://imgur.com/a/3imytkd)

**Comment by esmoak** [comment]
I will see if I can repeat this. I noticed that the times it happened it was on night mode, after sunset and before sunrise. Could be coincidence. 

I will record what I can the next time it happens. I've used it multiple times in day mode today and have had no issues.

**Comment by esmoak** [comment]
https://i.postimg.cc/FR9KLFVR/Screenshot-20190219-082452.jpg 
https://i.postimg.cc/262nqKvs/Screenshot-20190219-084146.jpg 
https://i.postimg.cc/XYCK2xS5/Screenshot-20190219-084053.jpg 
Here are some screenshots. Sorry, the screen recorder I tried sucked,
 so I just did screenshots. 

No amount of interacting allowed me to fully reveal the third alarm. When I exit the program I will sometimes get slightly more revealed. I
 tried expanding each alsrm, etc, there is definitely a glitch at least with with my phone

**Comment by esmoak** [comment]
Finally got a better demonstration with video: 

https://imgur.com/gallery/iSFrQa9

**Comment by fennifith** [comment]
Wow, I'm not sure what's happening here. It looks like the bottom sheet is getting stuck in a state that it shouldn't be in - maybe it's computing the available height incorrectly - but the fact that you can switch to the settings tab and back with no change somewhat debunks that theory. This might be more of an issue with the `material-components` dependency than Alarmio's implementation of it, especially considering that (so far) it only occurs on one device.

**Comment by esmoak** [comment]
Yeah, I don't know what is going on. I just know that the only fix seems to be a reinstall. 

I haven't reinstalled yet since posting that video clip and have rebooted a few times and the issue still persists. 

Thanks for looking into the issue.
